### DISCUSSION TITLE
Entry: Postgresql, Output: typesense | CDC not working

### DISCUSSION THREAD

**Discussion by florian-deroo** [airbytehq/airbyte#35398] [Connector Questions]
Hi everyone! I'm having some trouble when i'm trying to use CDC with the postgres connector. When i add new values to my table, the value is not sent to my output which is typesense. However, it's working when i'm doing a manual sync, so i guess i have misconfigured the CDC, here is my current config in postgresql.conf (i've restarted my postgres server)

wal_level = 'logical'
max_wal_senders = 1
max_replication_slots = 1

When i verify my config with the postgres requests:
SHOW max_wal_senders;
![image](https://github.com/airbytehq/airbyte/assets/59513489/ada972ac-f20c-4100-9ad0-03cecefb1565)
SHOW wal_level;
![image](https://github.com/airbytehq/airbyte/assets/59513489/69c7fb4b-ca2e-4516-838d-16068cc3cd27)
SHOW max_replication_slots ;
![image](https://github.com/airbytehq/airbyte/assets/59513489/f4ff687a-208c-4935-9897-bdb27eaf3de7)

Publication table:
SELECT * FROM pg_publication_tables;
![image](https://github.com/airbytehq/airbyte/assets/59513489/8f4a5640-547b-4bd0-82a7-711374f417a2)

Replication slots
SELECT * FROM pg_replication_slots;
![image](https://github.com/airbytehq/airbyte/assets/59513489/71803359-ad67-4ccd-962d-389e973e26cf)

On the previous screen we can see that active is false but i have no idea why?

Here is my CDC Config on postgresql:
![image](https://github.com/airbytehq/airbyte/assets/59513489/0f386817-076a-4cc7-8815-64e71d8568a9)

Do you have an idea why it's not working?

**Comment by florian-deroo** [comment]
UP

**Reply by marcosmarxm** [reply]
Hi @florian-deroo, apologies for the delayed response. I'll investigate the sync issue between the source and destination this week. In the meantime, could you please test the connection and try sending the data to a certified destination like PostgreSQL or BigQuery? This will help us pinpoint whether the issue is with the source or destination, and using a certified connector might reveal more about the root cause. Thanks!

**Comment by soumilbaldota** [comment]
This is still an issue for me.

[a comment by an automated account is not shown]
